uint32 rostime_secs
string source
uint32 severity		# 0 - INFO, 1 - WARN, 2 - ERROR, 3 - DEBUG
uint32 error_id
string message
string context
uint32 log_id

